AcquisitionInfo/explainer.md

```diff
@@ -265,6 +265,32 @@ the acquisition information that is recorded for the originally acquired applica
 other installations. This applies to any and all other installation for the same user profile across any other device supported
 by the UA.
 
+For example, a user may discover App A through an ad campaign run on the Microsoft Store. The user proceeds to install App A on
+Device A. Running `getDetails()` from the newly acquired app would return the following:
+
+```js
+details = {
+  installSource: "apps.microsoft.com",
+  attributionId: "adCampaign",
+  ...
+} 
+```
+
+The same user logs in on Device B and logs into the profile that installed App A on Device A. Sync would install App A once again
+but on Device B, at which point the `getDetails()` payload would return the following:
+
+```js
+details = {
+  installSource: "Sync",
+  attributionId: "adCampaign",
+  ...
+} 
+```
+
+While the `installSource` changes to correctly reflect the app's new install source, the `attributionId` stays the same. This
```

Comment:
I think it would be worthwhile to add an additional note after this paragraph to reinforce that it's not that we're specifically carrying forward a property named `attributionId`, but rather that we're carrying forward any properties that were recorded at the time of acquisition.

Additionally, different install sources may use the same property name but in different ways.  Ex:

Install Source A might add a property called `releaseDate` with a value of `2024-8-20` while Install Source B may also happen to have a same-named property that happens to use an epoch (`1724115600`).  Consumers of this info would likely know what properties to look for based on the `installSource`.  If the `installSource` suddenly gets changed to `Sync`, then they may lose the context of how to correctly parse/interpret that data.

Given that, while it makes sense to have `installSource` be `Sync` for these sync'd web apps, we may also want to have an `originalInstallSource` property for those sync'd apps which still references the original install source.

Reply:
Thanks for the review!
That sounds like a good alternative for keeping that case in mind. How do you feel about the possibility of adding a separate boolean property of `Sync` for sync'd web apps, which would let us maintain `installSource` as the original value and avoid similar verbiage that `originalInstallSource` might be confused with?

Reply:
That approach seems reasonable to me.  Can you put out an update to this PR with that change, and then we can gather additional feedback from @dmurph who originally posed the question?

Reply:
That sounds good to me as well.